internal/grafanadi/service/span.go

```diff
@@ -41,20 +52,48 @@ func ConverSpan2Frame(spans []*storagemodel.Span) model.DataFrame {
 		durationAry = []int64{int64(rootEnd.Sub(rootBegin) / 1000 / 1000)}
 	}
 
+	var opsType string
+	m1 := make(map[string]string, 0)
```

Comment:
The variable name of m1 needs to be optimized. It makes no sense to define it like this.